images/pre-provision-command/README.md

```diff
@@ -0,0 +1,44 @@
+# CAPH pre-provision-command
```

Comment:
I think you have to either reference this in the docs/ folder or put it into the docs folder

Reply:
I extended the docs and moved it to the devel-docs.